OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -237,6 +240,15 @@ $ sudo chown 1000:1000 /scratch/wcpdocker/volumes/es
 
 #### B. Creating and Running Elasticsearch Container
 
+>Note: With the beginning of April 2022 PSU release of Oracle WebCenter Portal, to install Elasticsearch you need to download the Elasticsearch 7.17.x binaries from Elasticsearch [Releases](https://www.elastic.co/downloads/past-releases#elasticsearch).
+
+Update the file name as `ELASTIC_SEARCH_INSTALLER_FILE_NAME` value and Elasticsearch version as `ELASTIC_SEARCH_VERSION` value.
+
+Copy the downloaded Elasticsearch binary to portal managed server using below command. 
+```
+$ docker cp elasticsearch-7.17.x-linux-x86_64.tar.gz <WCPORTLET_CONTAINER_NAME>:/u01/oracle/user_projects
```

Comment:
Isn't `/u01/oracle/user_projects` mapped to a host directory? If so, why not just copy this on the host instead of into the container?